### ISSUE TITLE
Evaluate code as a command

### ISSUE THREAD

**Issue by ccontour** [ccontour/steggy#154]
VMService provides all available local data already, along with some standard utilities for code to work with. With the help of configuration provided by home assistant's service listing, it should be possible to do a bit of code generation to build wrappers around the home assistant call service which correspond with all the commands. This can let code input on the UI have indirect  access to the apis that send messages to home assistant

- Also directly inject relevant providers such as socket/fetch api services 

## Notes

A bit of reference POC code: https://github.com/ccontour/steggy/blob/master/libs/home-assistant/src/domains/media-player-domain.service.ts

The current implementations aren't that helpful currently, but the function format seems fine for this type of dynamic work

Not directly dependent on #153, but would strongly improve the experience to have a real editor on hand to interact with

**Comment by ccontour** [comment]
Blocked by #157

**Comment by ccontour** [comment]
Unblocked via new dedicated breakout api. They will be the primary intended way from the vm to access services. Native providers may be added in the future, but as a lower priority since the breakout api can have new methods added as needed

**Comment by ccontour** [comment]
Appears in `0.12.9`
